src/AnisoPairEvaluatorTwoPatchMorse.h

```diff
@@ -2,37 +2,59 @@
 // Copyright (c) 2021-2024, Auburn University
 // Part of azplugins, released under the BSD 3-Clause License.
 
-/*!
- * \file AnisoPairEvaluatorTwoPatchMorse.h
- * \brief Defines the aniostropic pair force evaluator class for Two-patch Morse potential
- */
-
 #ifndef AZPLUGINS_ANISO_PAIR_EVALUATOR_TWO_PATCH_MORSE_H_
 #define AZPLUGINS_ANISO_PAIR_EVALUATOR_TWO_PATCH_MORSE_H_
 
 #include "AnisoPairEvaluator.h"
 
 #include "hoomd/VectorMath.h"
 
-#ifdef NVCC
+#ifdef __HIPCC__
 #define DEVICE __device__
 #define HOSTDEVICE __host__ __device__
 #else
 #define DEVICE
 #define HOSTDEVICE
 #endif
 
+namespace hoomd
+    {
 namespace azplugins
     {
-
 namespace detail
     {
 //! Two-patch Morse parameters
-/*!
- * \sa AnisoPairEvaluatorTwoPatchMorse
- */
-struct two_patch_morse_params : public AnisoPairParams
+struct AnisoPairParametersTwoPatchMorse : public AnisoPairParameters
     {
+#ifndef __HIPCC__
+    AnisoPairParametersTwoPatchMorse()
+        : Mdeps(0), Mrinv(0), req(0), omega(0), alpha(0), repulsion(false)
```

Comment:
Since `Mrinv` is an inverse quantity, it can't be zero... let's default it 1.0.
```suggestion
        : Mdeps(0), Mrinv(1.0), req(0), omega(0), alpha(0), repulsion(false)
```